charts/atlas-operator/templates/crds/crd.yaml

```diff
@@ -442,6 +442,40 @@ spec:
           spec:
             description: AtlasSchemaSpec defines the desired state of AtlasSchema
             properties:
+              cloud:
+                description: Cloud defines the Atlas Cloud configuration.
+                properties:
+                  project:
+                    type: string
+                  tokenFrom:
```

Comment:
Use "repo"

Reply:
I don't think "project" is actually used or necessary in AtlasMigration.

Reply:
Yep, I'll do rename